examples/morpheusvm/tests/workload/genesis.go

```diff
@@ -68,31 +66,20 @@ func newDefaultKeys() []ed25519.PrivateKey {
 		}
 		testKeys[i] = ed25519.PrivateKey(bytes)
 	}
-
 	return testKeys
 }
 
-type NetworkConfiguration struct {
-	workload.DefaultTestNetworkConfiguration
-	keys []ed25519.PrivateKey
-}
-
-func (n *NetworkConfiguration) Keys() []ed25519.PrivateKey {
-	return n.keys
-}
-
-func NewTestNetworkConfig(minBlockGap time.Duration) (*NetworkConfiguration, error) {
+func NewTestNetworkConfig(minBlockGap time.Duration) (workload.DefaultTestNetworkConfiguration, error) {
 	keys := newDefaultKeys()
 	genesis := newGenesis(keys, minBlockGap)
 	genesisBytes, err := json.Marshal(genesis)
 	if err != nil {
-		return nil, err
+		return workload.DefaultTestNetworkConfiguration{}, err
 	}
-	return &NetworkConfiguration{
-		DefaultTestNetworkConfiguration: workload.NewDefaultTestNetworkConfiguration(
-			genesisBytes,
-			consts.Name,
-			vm.NewParser(genesis)),
-		keys: keys,
-	}, nil
+	return workload.NewDefaultTestNetworkConfiguration(
+		genesisBytes,
+		consts.Name,
+		vm.NewParser(genesis),
+		keys,
+	), nil
```

Comment:
Why change from pointer to value here?

Reply:
It is because NetworkConfiguration embedded workload.DefaultTestNetworkConfiguration value. Do you want to switch to a pointer?